src/main/scala/akka/wamp/Wamp.scala

```diff
@@ -52,7 +61,12 @@ object Wamp extends ExtensionId[WampExtension] with ExtensionIdProvider {
     * @param url is the URI to connect to (e.g. "ws://somehost.com:9999/path/to/ws")
     * @param subprotocol is the WebSocket subprotocol to negotiate (e.g. "wamp.2.msgpack" or  "wamp.2.json")
     */
-  final case class Connect(client: ActorRef, url: String = "ws://localhost:8080/ws", subprotocol: String = "wamp.2.json") extends Command
+  final case class Connect(client: ActorRef,
+                           url: String = "ws://localhost:8080/ws",
+                           subprotocol: String = "wamp.2.json",
+                           messageFlowDecider: Supervision.Decider = {
```

Comment:
Please read last comment I wrote on closing #20 

You'll find almost what you were looking for from release [v0.8.0](https://github.com/angiolep/akka-wamp/tree/v0.8.0) onwards